pkg/ingester/otlp/convert.go

```diff
@@ -202,8 +203,11 @@ func (p *profileBuilder) convertLocationBack(ol *otelProfile.Location) uint64 {
 	if i, ok := p.locationMap[ol]; ok {
 		return i
 	}
-	lmi := ol.MappingIndex_.(*otelProfile.Location_MappingIndex)
-	om := p.src.MappingTable[lmi.MappingIndex]
+	var lmi int32 = 0
+	if ol.MappingIndex != nil {
```

Comment:
What happens if it is nil? I am not sure there is a correct mapping placeholder at index 0. Maybe we should fail the conversion in this case?

Reply:
Panic, I guess before it would also have paniked. 

Handled the error now in dfe456645